tutorials/clinical/30_ecog.py

```diff
@@ -181,12 +181,12 @@
 
 xyz_pts = np.array([dig['r'] for dig in evoked.info['dig']])
 
-src = mne.read_source_spaces(
-    op.join(subjects_dir, 'fsaverage', 'bem', 'fsaverage-ico-5-src.fif'))
+# use `src=None` for default source space, to specify spacing use
+# `src = mne.setup_source_space('fsaverage', subjects_dir=subjects_dir,
+#                               spacing='ico5', surface='pial')`
```

Comment:
... then you don't actually need to make these changes because `pial` rrs will be used